### PR TITLE
frontend: Proxy flatpak.org

### PR DESCRIPTION
**Pull request by dbnicholson**
Proxy the flatpak.org site from front.flathub.org like all of the
flathub.org sites. Primarily this is for consistency so that the
frontend servers are truly the frontend for all websites. However, it
clearly will provided a better experience for users.

The nginx configuration is mostly copied from webroot. Note that
default.d snippets are not used because that would interfere with
default including all of the snippets since they currently represent
different flathub paths.

Fixes flathub/flathub#707.

### CODE REVIEW

File: roles/frontend/templates/nginx-conf.d-flatpak-ssl.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    # OCSP Stapling ---
+    # fetch OCSP records from URL in ssl_certificate and cache them
+    ssl_stapling on;
+    ssl_stapling_verify on;
+
+    ## verify chain of trust of OCSP response using Root CA and Intermediate certs
+    ssl_trusted_certificate /var/lib/certsync/certs/{{ frontend_nginx_flatpak_domain }}/chain.pem;
+
+    resolver 93.93.128.2 93.93.130.2;
```
Comment (ramcq) [inline roles/frontend/templates/nginx-conf.d-flatpak-ssl.conf.j2]: There's a variable for these - frontend_nginx_resolver defined in the mythic group.

**Review by ramcq — CHANGES_REQUESTED** [review]
Looks good, one nitpick.

File: roles/frontend/templates/nginx-conf.d-flatpak-ssl.conf.j2 (hunk 1 above)
Comment (dbnicholson) [reply roles/frontend/templates/nginx-conf.d-flatpak-ssl.conf.j2]: Heh. It's hardcoded all over the place. I'll see about fixing that.

Comment (ramcq) [reply roles/frontend/templates/nginx-conf.d-flatpak-ssl.conf.j2]: WHAT...?! WHO WOULD EVEN DO THAT?!

:innocent: 

Thanks.

Comment (dbnicholson) [reply roles/frontend/templates/nginx-conf.d-flatpak-ssl.conf.j2]: It was just 2 other spots, but addressed now.

**Review by ramcq — APPROVED** [review]

**Pull request merged**